### ISSUE TITLE
Is this the kernel that finally works?

### ISSUE THREAD

**Issue by SteveBetter** [alef78/x86_ramos_i9_kernel#1]
I built this kernel with ubuntu 22.04 successfully after dealing with some building errors. After downloading the boot.bin in my ramos i9, the pad stays with the i9 logo for a few seconds, then reboots, after reboots for several times, it enters in recovery mode.( the process of making boot.bin is verified)
Has anyone succeeded in getting this kernel running?

**Comment by alef78** [comment]
Yes, it worked on my device. Kernel need to be signed before flashing to device.
Signing key and my utilities for signing can be found is this repository https://github.com/alef78/intel-ifwi-study , key.pem was downloaded from some Chinese site. 
Note, the device is very old, I worked on it about 10 years ago and I do not remember all details.

You may find some information in this thread https://4pda.to/forum/index.php?showtopic=499488&st=2460
(it is in Russian, but you may use google translate).
Some people built custom firmware (with android 5) for Ramos i9, based on drivers from this kernel.

Also note, this kernel is for original Ramos i9 tablet, which does not have 3G.
There is 3G version with different hardware and different signing key (which is probably unpublished, so this kernel works only on non-3G version).

**Comment by SteveBetter** [comment]
Thanks for your reply!
My pad doesn't have 3G, and packing boot.bin with your utilities has been verified on the official img file(an address offset needs to be modified). The device boots with the repacked official boot.bin.
I also tried the asus zenfone 5 kernel (https://github.com/zf5/android_kernel_asus_T00F). The device reboots for several times and then the screen gets black. lsusb gets nothing. dmesg gets something about usb:

[ 8208.869330] usb 1-3: new high-speed USB device number 20 using xhci_hcd
[ 8209.030768] usb 1-3: New USB device found, idVendor=8086, idProduct=09ee, bcdDevice= 0.a0
[ 8209.030786] usb 1-3: New USB device strings: Mfr=2, Product=1, SerialNumber=3
[ 8209.030794] usb 1-3: Product: CLOVERVIEW
[ 8209.030801] usb 1-3: Manufacturer: INTEL
[ 8209.030807] usb 1-3: SerialNumber: D62D6AAB7C0C4D30
[ 8209.035661] usb 1-3: USB disconnect, device number 20

Is it possible that it relates to the gcc version?
I am using "gcc version 11.4.0 (Ubuntu 11.4.0-1ubuntu1~22.04)"
Is there an old version stand-alone gcc?

**Comment by SteveBetter** [comment]
I tried it with gcc-4.9 in a docker container. Didn't boot up, either.

**Comment by SteveBetter** [comment]
Hi, @alef78
Would you like to compile this kernel again? If you don't have the device anymore, I can test it with my device.
Thanks very much.
